dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerArguments.java

```diff
@@ -62,7 +60,7 @@
   private final CommandLine commandLine;
   private ImmutableList<Path> cachedInputClasspath;
   private ImmutableList<Artifact> cachedArtifacts;
-  private ImmutableMap<Path, Artifact> pathToArtifact;
+  private ImmutableMap<Path, Artifact> cachedPathToArtifact;
```

Comment:
Here's another one. We have two lists and a map to track what is conceptually one collection.